### PR TITLE
Release v0.9-rc2

### PR DESCRIPTION
**Pull request by khs1994**
### Proposed changes (Mandatory)

Add `docker config` Only Swarm mode #283 

Add `LinuxKit` #287 

### Fix issues (Optional)

Remove useless file 

error in `SUMMARY.md`

### CODE REVIEW

**Comment by khs1994** [comment]
WaitFish 的《Docker学习手册v1.0》已归档到新的分支

File: _local/.bashrc_docker [hunk 1]
```diff
@@ -1,49 +0,0 @@
-# Some useful commands to use docker.
-# Author: yeasy@github
-# Created:2014-09-25
```
Comment (yeasy) [inline _local/.bashrc_docker]: Would like to suggest keep this file, as we wanna add some quick cmd alias if possible.

File: appendix_command/README.md [hunk 2]
```diff
@@ -1,126 +0,0 @@
-# Docker命令查询
```
Comment (yeasy) [inline appendix_command/README.md]: Why need to delete this file? Instead, we should update its content, for quick cmd usage reference.

File: _local/.bashrc_docker (hunk 1 above)
Comment (khs1994) [reply _local/.bashrc_docker]: 该文件不是用于 进入容器一节的吗，进入容器一节已经不需要该文件了

File: appendix_command/README.md (hunk 2 above)
Comment (khs1994) [reply appendix_command/README.md]: https://github.com/yeasy/docker_practice/tree/master/appendix/command

与这个文件重复了，SUMMARY 包含的是这个，没包含更目录的 appendix_command/README.md

File: _local/.bashrc_docker (hunk 1 above)
Comment (khs1994) [reply _local/.bashrc_docker]: https://github.com/yeasy/docker_practice/blob/v0.8/container/enter.md

Comment (khs1994) [reply _local/.bashrc_docker]: 没必要弄些额外的脚本，尽量用 docker 原始命令

**Pull request merged**